server/models/entity_comment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Entity_Comment = sequelize.define('Entity_Comment', {
+    comment: DataTypes.TEXT
+  }, {});
+  Entity_Comment.associate = function (models) {
+    Entity_Comment.belongsTo(models.Entity, { foreignKey: "entity_id" });
+    Entity_Comment.belongsTo(models.User, { foreignKey: "user_id" });
```

Comment:
Entity_Comment.belongsTo(models.User, { foreignKey: "user_id" }); indicates that there is an association between the User & Entity_Comment ? 

Meaning that this is a join table that holds both user_id & entity_id?

Reply:
Entity_Comment is more specifically, a 'Comment' on an 'Entity'. Sorry for the confusing naming, Entity_Comment is not meant as a join table. I should probably rename it. I was just following the diagram here: https://docs.google.com/document/d/1Qrq4zKWv4ojhY300WW26UfULMh4TYPoNKzjQNMNqMNY/edit

A Comment is supposed to only have one User, hence line 8 (and lack of assoc in User model)
A Comment is only 'commented' under one Entity as well, hence line 7

Not exactly a join table, just assocs to both User and Entity

Reply:
ah my bad, I think this comment was suppose to be for the liked_entity.js

Reply:
LOL xD still don't quite get this haha maybe we can ask Shums || Bonnie || Aceio?